tests/drum/test_fit.py

```diff
@@ -122,10 +122,32 @@ def test_fit_for_use_output_and_nested(
         )
 
     @pytest.mark.parametrize(
-        "framework", [RDS, SKLEARN_BINARY, SKLEARN_REGRESSION, XGB, KERAS, PYTORCH]
+        "framework, problem, docker",
+        [
+            (RDS, BINARY_TEXT, None),
+            (RDS, REGRESSION, None),
+            (RDS, ANOMALY, None),
+            (RDS, MULTICLASS, None),
+            (SKLEARN_BINARY, BINARY_TEXT, DOCKER_PYTHON_SKLEARN),
+            (SKLEARN_REGRESSION, REGRESSION, DOCKER_PYTHON_SKLEARN),
+            (SKLEARN_BINARY, BINARY_TEXT, None),
```

Comment:
why don't you have docker for multiclass / anomaly?

Reply:
I unrolled the cases that were already happening. I can add that case if you'd like, but didn't think it necessary